frontend/src/components/Psychologist/Appointments/ConventionModal.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { observer } from 'mobx-react';
+import { Modal, ModalTitle, ModalContent } from '@dataesr/react-dsfr';
+import { useStore } from 'stores/';
+import { checkConvention } from 'services/conventionVerification';
+import ConventionForm from '../Reimbursement/ConventionForm';
+
+const ConventionModal = ({ currentConvention }) => {
+  const { commonStore: { config } } = useStore();
+  const [isOpen, setIsOpen] = useState(true);
```

Comment:
Is `isOpen` needed, since this component is not render when conditions doesn't match (see Appointment.jsx l.67-71)?

Reply:
it's smoother to close it manually because the convention can take time to update :/
so not really needed but a bit better